src/enclave/Enclave/ExpressionEvaluation.h

```diff
@@ -3,8 +3,11 @@
 #include <functional>
 #include <typeinfo>
 #include <cmath>
+// #include <ctime>
 #include <limits>
 
+#include <iostream>
```

Comment:
Can you remove this additional (unnecessary?) include?

Reply:
done!